src/http2_connection.erl

```diff
@@ -835,7 +835,10 @@ route_frame(F={H=#frame_header{stream_id=StreamId}, #window_update{}},
               }}
 
     end;
-
+route_frame({#frame_header{type=T}, _}, Conn)
+  when T > ?CONTINUATION ->
+    lager:debug("Ignoring Unsupported Expansion Frame"),
+    {next_state, connected, Conn};
```

Comment:
For now, because all this is really saying is that frame types 10 and higher are reserved for future RFCs, but we shouldn't explode if some client that supports them starts sending them to us